mvn build failed (501 HTTPS Required)

Following build error:

```
[ERROR] Failed to execute goal on project luyten: Could not resolve dependencies for project us.deathmarine:luyten:jar:0.5.4: Failed to collect dependencies at org.bitbucket.mstrobel:procyon-core:jar:0.5.36: Failed to read artifact descriptor for org.bitbucket.mstrobel:procyon-core:jar:0.5.36: Could not transfer artifact org.bitbucket.mstrobel:procyon-core:pom:0.5.36 from/to central (http://repo.maven.apache.org/maven2): Failed to transfer file http://repo.maven.apache.org/maven2/org/bitbucket/mstrobel/procyon-core/0.5.36/procyon-core-0.5.36.pom with status code 501 -> [Help 1]
```

You need to change all ```http://``` to ```https://``` in *pom.xml*, then build is OK :)

--- a/pom.xml
+++ b/pom.xml
@@ -1,5 +1,4 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>us.deathmarine</groupId>
 	<artifactId>luyten</artifactId>
@@ -50,7 +49,7 @@
 			</snapshots>
 			<id>central</id>
 			<name>Central Repository</name>
-			<url>http://repo.maven.apache.org/maven2</url>
+			<url>https://repo.maven.apache.org/maven2</url>
 		</repository>
 	</repositories>
 	<build>

